### ISSUE TITLE
gopls: automated issue report (initialization)

### ISSUE THREAD

**Issue by angelhvargas** [golang/vscode-go#1024]
gopls version: v0.5.5
gopls flags: 

ATTENTION: PLEASE PROVIDE THE DETAILS REQUESTED BELOW.

Describe what you observed.

<ANSWER HERE>

<pre>Starting client failed
  Message: unsupported URI schemes: [{vsls:/ AOC-2018-7}] (gopls only supports file URIs)
  Code: 0 </pre>

OPTIONAL: If you would like to share more information, you can attach your complete gopls logs.

NOTE: THESE MAY CONTAIN SENSITIVE INFORMATION ABOUT YOUR CODEBASE.
DO NOT SHARE LOGS IF YOU ARE WORKING IN A PRIVATE REPOSITORY.

<OPTIONAL: ATTACH LOGS HERE>

**Comment by hyangah** [comment]
@angelhvargas Thanks for the report. Can you please tell us what's the extension version? The latest versions of the extension should recognize the VSCode LiveShare.

**Comment by angelhvargas** [comment]
Hello sure, seems to be v0.5.5

**Comment by hyangah** [comment]
@angelhvargas I meant the extension version. You can check the version info from the extension view.
The latest version is v0.19.1.

![Screen Shot 2020-12-11 at 9 46 37 AM](https://user-images.githubusercontent.com/4999471/101917261-09c57200-3b96-11eb-945a-fa88956cf007.png)

**Comment by angelhvargas** [comment]
Thanks for clarify. Currently I have version 0.19.1

**Comment by hyangah** [comment]
Thank you so much for following up. @angelhvargas 
I don't know what's changed between then (I thought I fixed the issue) and now. 
I just verified by running a liveshare session with another person and it doesn't pop up the annoying message any more.

Still the liveshare experience as guests is limited - you will not be able to run, debug, build, but that's probably a harder problem. 

Will release the fix early next week. By then, either disable the language server when  you join liveshare or, even disable the extension. The intellisense will come from the host's machine, anyway.
